docs/source/changelog.rst

```diff
@@ -2,31 +2,6 @@
 Changelog
 *********
 
-Changes in main
```

Comment:
Should we add this back in? In theory it's a good place for developers to write down and accumulate changes for the next release version, although in practice I don't know how often it get used?

Reply:
no, we should only include the singular last commit related to the typo fix.

Reply:
Earl's commits for release should not be included.